qa/workunits/rbd/cli_generic.sh

```diff
@@ -1387,6 +1387,7 @@ test_mirror_snapshot_schedule() {
     done
     rbd mirror snapshot schedule status | grep 'rbd2/ns1/test1'
 
+    # set a global level mirror image snapshot schedule
```

Comment:
Can we also cover snapshot schedules at multiple levels like pool, namespace, group and image level,

also if user tries all these things does it take precedence over another schedule?

Reply:
This line is not part of what is under review in this PR.  We are only concerned with Vinay's [librbd::get_group_snap_get_mirror_namespace API and groups status and health in mirror pool status](https://github.com/ceph/ceph/pull/62041/commits/d818362d011938c8a82d6408f339fd06217a311b) commit here.  Comments on everything else should be posted in https://github.com/ceph/ceph/pull/53793 (the "main" PR).

Reply:
Ok please make sure this coves in main PR